From 13a868fb8d1ab7c1948d6d5b2122cad778e75f4f Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Fri, 25 Nov 2022 09:54:05 +0200
Subject: [PATCH] [SW-112010] greco: correct bootinfo offset

This patch corrects bootinfo offset in spsram
for greco to 0xC0000.

Change-Id: I1c8725f302bc7a5734bfea401dd3173ddf31b41f
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/greco/arc/common.h b/arch/greco/arc/common.h
index a9792f1..2a1ed28 100644
--- a/arch/greco/arc/common.h
+++ b/arch/greco/arc/common.h
@@ -28,12 +28,8 @@
 
 #define FW_WORD_NUM		0x2200	/* FW_SIZE / 2 */
 
-
-/*
- * TODO: this address is wrong, a good address must be set in order to use
- * boot info API.
- */
-#define BOOT_INFO_BASE		SP_SRAM_BASE
+#define BOOT_INFO_OFFSET	0xC000
+#define BOOT_INFO_BASE		(SP_SRAM_BASE + BOOT_INFO_OFFSET)
 
 #define FUSE_WORDS_PER_BANK	0x40
 #define I2C_SLAVE_ADDRESS	0x6A
